BaseLib/BuildInfo.h

```diff
@@ -32,6 +32,7 @@ namespace BuildInfo
 
     extern BASELIB_EXPORT const std::string git_describe;
     extern BASELIB_EXPORT const std::string ogs_version;
+    extern BASELIB_EXPORT const std::string cmake_args;
```

Comment:
Btw. BASELIB_EXPORT seems to be an empty macro. Can't we just remove it?

Reply:
I thought this was required for shared libs on Windows (.dlls)?

Reply:
Yes, probably. I got confused. After some googling I now roughly understand how this works.
https://cmake.org/cmake/help/v3.0/module/GenerateExportHeader.html